DataFormats/Detectors/TPC/src/ClusterNativeHelper.cxx

```diff
@@ -277,3 +285,153 @@ void ClusterNativeHelper::TreeWriter::close()
   mTree.release();
   mFile.reset();
 }
+
+uint64_t ClusterNativeHelper::readClusterNativeInWorkflow(ClusterNativeAccess& clusterIndex, std::unique_ptr<ClusterNative[]>& clusterBuffer, MCLabelContainer& clusterMCBuffer, o2::framework::ProcessingContext& pc, bool processMC, gsl::span<const o2::tpc::Digit>* inputDigits, std::unique_ptr<const MCLabelContainer>* inputDigitsMC)
```

Comment:
This is basically the input handling from the CA tracker and I think that is first of all the wrong place and I really would like to keep this to ClusterNative related stuff, it's the wrong place for handling digits.

Reply:
I agree on moving it somewhere else because it seems unrelated. But there was so much copy and pasted code, so if we can, I'd keep digits and cluster reading together.